docs/api/menu.md

```diff
@@ -59,18 +59,18 @@ will become properties of the constructed menu items.
 
 The `menu` object has the following instance methods:
 
-#### `menu.popup([browserWindow, options, callback])`
+#### `menu.popup(options)`
```

Comment:
This refactor gets rid of so much ambiguous, confusing code. Enthusiastic :+1: on moving all the optional arguments into the `options` object.